File: hw/cevallos/u2/hw19FarmSystem/Python/Cow.py
from FarmAnimal import FarmAnimal

class Cow(FarmAnimal):
    def __init__(self, is_producing_milk, liters_a_day, id, breed, born_on, gender, is_able_to_reproduce, weight, cage):
        super().__init__(id, breed, born_on, gender, is_able_to_reproduce, weight, cage)
        self.is_producing_milk = is_producing_milk
        self.liters_a_day = liters_a_day
    
    def milk(self):
        self.liters_a_day += 1
        return self.liters_a_day
    
    def __str__(self):
        return f"Cow{{isProducingMilk={self.is_producing_milk}, litersADay={self.liters_a_day}, {super().__str__()}}}"

File: hw/cevallos/u2/hw19FarmSystem/JavaScript/Cow.js
const FarmAnimal = require('./FarmAnimal');

class Cow extends FarmAnimal {
    constructor(isProducingMilk, litersADay, id, breed, bornOn, gender, isAbleToReproduce, weight, cage) {
        super(id, breed, bornOn, gender, isAbleToReproduce, weight, cage);
        this.isProducingMilk = isProducingMilk;
        this.litersADay = litersADay;
    }

    milk() {
        this.litersADay++;
        return this.litersADay;
    }

    toString() {
        return `Cow{isProducingMilk=${this.isProducingMilk}, litersADay=${this.litersADay}, ${super.toString()}}`;
    }
}

module.exports = Cow;